Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -601,28 +646,51 @@ def fetch_incidents():
     # Handle first time fetch, fetch incidents retroactively
     if last_fetch is None:
         last_fetch, _ = parse_date_range(FETCH_TIME, date_format='%Y-%m-%dT%H:%M:%SZ')
-    last_fetch_timestamp = date_to_timestamp(last_fetch, date_format='%Y-%m-%dT%H:%M:%SZ')
+
+    # timestamp creation excepts only '%Y-%m-%dT%H:%M:%SZ' date format
+    if '.' in last_fetch:
```

Comment:
The issue this fixes is that the fact that milliseconds failed the date_to_timestamp method because it had trouble parsing it. Removing the milliseconds solved the issue (I did try adding the %f flag to the format, that did not work either).

As for timezone issues - it seems the query to falconstrike works on a local time (same goes for what I've seen in the ADP laptop) so I don't believe there are timezone differences.